terraform/Makefile

```diff
@@ -0,0 +1,35 @@
+.DEFAULT_GOAL := provision
+
+.PHONY: checks
+checks:
+ifndef TERRAFORM_PROJECT
+	$(error TERRAFORM_PROJECT is undefined)
+endif
+ifndef PR_NUMBER
+	$(error PR_NUMBER is undefined)
+endif
```

Comment:
This Makefile is common for all Terraform "entry projects" to create/destroy resources, so it needs to receive the `TERRAFORM_PROJECT` and `PR_NUMBER` environment variables. Each Terraform entry project will have its own Terraform state file pointing to S3. Note that for this initiative, we will have only 2 entry projects:
- The one used to spin up/down SUSE builder VMs, needed to build the Fluent Bit executable for these OSes.
- The one used to spin up/down all the VMs to run the E2E tests